### DISCUSSION TITLE
[UdeA Students] Estudiantes con cursos duplicados

### DISCUSSION THREAD

**Discussion by Felipe-RA** [rramosp/ai4eng.v1#126] [UDEA Students]
Saludos estudiantes

Este post está enfocado en estudiantes de la UdeA que tienen cursos duplicados
(cancelaron en algún momento/tienen múltiples cursos)

Por favor poner aquí abajo el correo vuestro **correo institucional**

Gracias!

**Comment by ebm7pc** [comment]
Correo:
ariel.bedoya@udea.edu.co 
Tengo suscripciones duplicadas porque matriculé el curso en semestres anteriores pero cancelé.

**Comment by AlexGCas** [comment]
alexander.garciac@udea.edu.co

**Comment by mateoys** [comment]
mateo.yepes@udea.edu.co
tengo cursos duplicados , porque matriculé esta materia en el 2020 y la cancelé

**Comment by carlosmding** [comment]
carlos.pinto@udea.edu.co
Duplicado porque cancelé curso en 2021-2

**Comment by sofiagutz** [comment]
sofia.gutierrez@udea.edu.co
Me matriculé hace 1 semestre pero cancelé.

**Comment by valentinaechavarria** [comment]
valentina.echavarria1@udea.edu.co
Me matriculé el semestre pasado pero cancelé.

**Comment by efsiul** [comment]
felipe.cadavid@udea.edu.co
Me matricule hace 2 semestres pero cancele

**Comment by rramosp** [comment]
os acabo de actualizar la cuenta a todos los que habéis manifestado el problema.

si entráis, ahora sólo deberíais de ver la sesión de la materia correspondiente a este semestre

hacednos saber si todavía tenéis inconveniente

**Comment by Moorea-AI** [comment]
Hola Felipe, mi correo es aluz.moreno@udea.edu.co 
Tengo el curso duplicado porque cancelé empezando el semestre pasado.

Muchas gracias,

**Comment by rramosp** [comment]
ya también lo tienes reparado

On Tue, May 17, 2022 at 4:08 PM Moorea-AI ***@***.***> wrote:

> Hola Felipe, mi correo es ***@***.***
> Tengo el curso duplicado porque cancelé empezando el semestre pasado.
>
> Muchas gracias,
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/126#discussioncomment-2770904>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZINRQGCBTYIDFSB65N3VKQDD7ANCNFSM5WFUPZCA>
> .
> You are receiving this because you commented.Message ID: <rramosp/ai4eng.
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Reply by Moorea-AI** [reply]
Profe muchas gracias!

**Comment by Momos20** [comment]
Correo: 
camilo.catano@udea.edu.co
Tengo el curso duplicado porque cancelé  en el semestre 2020-1

**Comment by MariaCamilaLM** [comment]
mariac.lopera@udea.edu.co 

Tengo suscripciones duplicadas porque matriculé el curso el semestre anterior pero cancelé.

**Comment by rramosp** [comment]
ya lo debes de tener ok. verifica porfa

On Tue, May 17, 2022 at 8:25 PM Momos20 ***@***.***> wrote:

> Correo:
> ***@***.***
> Tengo el curso duplicado porque cancelé comenzando en el semestre 2020-1
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/126#discussioncomment-2771854>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIKDB6BENDN36TXDVZTVKRBKJANCNFSM5WFUPZCA>
> .
> You are receiving this because you commented.Message ID: <rramosp/ai4eng.
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Comment by andresfcm97** [comment]
Buenas noches, mi correo es felipe.ceron@udea.edu.co y tengo repetido el curso porque lo inicié hace unos días, ya que un compañero me compartió el link del curso.

**Comment by EstebanMa12** [comment]
desteban.maya@udea.edu.co
Tengo la asignatura duplicada porque matriculé el curso hace dos semestres y la cancele

**Comment by samle98** [comment]
samuel.lopera@udea.edu.co
Tengo el curso duplicado ya que cogí el curso en semestres anteriores pero cancelé.

**Comment by Andresfga** [comment]
andresf.grajales@udea.edu.co
Tengo el curso duplicado ya que el semestre pasado tuve que cancelarla.

**Comment by rramosp** [comment]
ya están eliminadas las sesiones anteriores del curso en los que habéis manifestado esa duplicidad hasta ahora

porfa verificadlo y hacednos saber acerca de cualquier inquietud

**Comment by estebanquingo12** [comment]
Buenos días. 
Me matriculé hace dos semestres pero cancelé. 
esteban.quinterog@udea.edu.co

**Comment by rramosp** [comment]
esteban, tienes que darnos tu correo udea

On Wed, May 18, 2022 at 8:09 AM estebanquingo12 ***@***.***>
wrote:

> Buenos días.
> Me matriculé el semestre pasado pero cancelé.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/126#discussioncomment-2775819>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIO4X4VPXRTHIMET7I3VKTTXBANCNFSM5WFUPZCA>
> .
> You are receiving this because you commented.Message ID: <rramosp/ai4eng.
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Reply by estebanquingo12** [reply]
Si profesor, es este: esteban.quinterog@udea.edu.co
 
Quedo atento.

**Comment by santiagoramirez10** [comment]
Buen día, me matriculé el semestre pasado (2021-2) pero cancelé, mi correo es santiago.ramirez10@udea.edu.co. Muchas gracias.

**Comment by juanitafonn** [comment]
juanita.fonnegra@udea.edu.co

Matriculé el curso el semestre pasado, pero por razones personales tuve que cancelar y matricularlo de nuevo éste semestre.

**Comment by andresqb198** [comment]
andres.quintero8@udea.edu.co

Me inscribí en el curso abierto.

**Comment by 53rv1d0r** [comment]
Hola! manuel.restrepog@udea.edu.co curso duplicado porque cancelé la materia el semestre anterior

**Comment by sfh9812** [comment]
Hola, santiago.francoh@udea.edu.co.
Me aparece el curso duplicado porque matriculé el curso semestres anteriores y la cancelé.

**Comment by Nilson16** [comment]
nilson.suarez@udea.edu.co
Matriculé en semestre anterior pero cancelé

**Comment by lauratobon** [comment]
yo tengo duplicado circuitos, pero no recuerdo si antes, cuanto pertenecía a la ingeniería en electrónica, alguna vez la cancelé ya que fue hace bastante tiempo. gracias

**Reply by lauratobon** [reply]
@lcecilia.tobon@udea.edu.co

**Comment by Felipe-RA** [comment]
> Buenas noches, me aparecen dos cursos en la plataforma y desde el colab no me deja avanzar la entrada. Mi correo es: [jhon.longas1@udea.edu.co](mailto:jhon.longas1@udea.edu.co)

**Comment by AlexBedoya0803** [comment]
Correo: alexander.bedoya1@udea.edu.co
Cancelé el semestre pasado

**Comment by rramosp** [comment]
ya está

On Mon, May 23, 2022 at 10:27 PM AlexBedoya0803 ***@***.***>
wrote:

> Correo: ***@***.***
> Cancelé el semestre pasado
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/126#discussioncomment-2808680>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZINIG24SGRCRUXOACOTVLREBNANCNFSM5WFUPZCA>
> .
> You are receiving this because you commented.Message ID: <rramosp/ai4eng.
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Reply by AlexBedoya0803** [reply]
Cuando entró al enlace del formulario que nos envió por correo me aparece el siguiente mensaje:
![image](https://user-images.githubusercontent.com/90013837/169944410-6cec9e3e-703e-4c5b-bb54-052d08313cd9.png)

**Comment by rramosp** [comment]
sigue las instrucciones en la página del curso. el enlace al formulario
está actualizado. pon atención a registrarte en el formulario de la materia
UdeA 2022.1, no en el de la versión abierta

On Mon, May 23, 2022 at 10:44 PM AlexBedoya0803 ***@***.***>
wrote:

> Cuando entró al enlace del formulario que nos envió por correo me aparece
> el siguiente mensaje:
> [image: image]
> <https://user-images.githubusercontent.com/90013837/169944410-6cec9e3e-703e-4c5b-bb54-052d08313cd9.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/126#discussioncomment-2808739>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZINSKFBW2LC6JPYS46LVLRGABANCNFSM5WFUPZCA>
> .
> You are receiving this because you commented.Message ID: <rramosp/ai4eng.
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Reply by AlexBedoya0803** [reply]
Listo, profe. Ya está.

¡Muchas gracias!

**Comment by EmiroMS** [comment]
Hola, hice el registro y me aparece el curso con fecha del semestre pasado
![image](https://user-images.githubusercontent.com/55450818/169944928-6366d2e4-635f-4c82-a066-0dce55181d93.png)

**Comment by rramosp** [comment]
danos tu correo

On Mon, May 23, 2022 at 10:49 PM EmiroMS ***@***.***> wrote:

> hola, apenas hice el registro y me aparece el curso del semestre pasado el
> cual cancele
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/126#discussioncomment-2808756>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIMKVG7HYBU7JZMFDWDVLRGT7ANCNFSM5WFUPZCA>
> .
> You are receiving this because you commented.Message ID: <rramosp/ai4eng.
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Comment by JuanCastaneda55** [comment]
buenos días, tengo el curso duplicado
Juan.casteneda@udea.edu.co
gracias.
![image](https://user-images.githubusercontent.com/106169733/170068243-283e1264-9c77-4afe-962a-d6b6128f4d42.png)

**Reply by Kristoferbuitrago** [reply]
Hola, presento el mismo problema, el curso está duplicado.
![curso_curso duplicado](https://user-images.githubusercontent.com/33135386/170407670-d6e1e396-147d-4f55-a093-f287cfa4f251.JPG)

**Comment by rramosp** [comment]
ya está

On Tue, May 24, 2022 at 9:55 AM JuanCastaneda55 ***@***.***>
wrote:

> buenos días, tengo el curso duplicado
> ***@***.***
> gracias.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/126#discussioncomment-2812867>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIKB5YDHNENYHFTMRRDVLTUW7ANCNFSM5WFUPZCA>
> .
> You are receiving this because you commented.Message ID: <rramosp/ai4eng.
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Comment by rramosp** [comment]
ya lo tienes arreglado

On Wed, May 25, 2022 at 10:08 PM Kristofer ***@***.***> wrote:

> Hola, presento el mismo problema, el curso está duplicado.
> [image: curso_curso duplicado]
> <https://user-images.githubusercontent.com/33135386/170407670-d6e1e396-147d-4f55-a093-f287cfa4f251.JPG>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/126#discussioncomment-2824377>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIPLYEUNUN44INFLH6DVL3TK7ANCNFSM5WFUPZCA>
> .
> You are receiving this because you commented.Message ID: <rramosp/ai4eng.
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Comment by Jhon2507** [comment]
Yo no había visto la materia antes y me salen dos cursos
Correo: jhon.longas1@udea.edu.co

**Reply by Felipe-RA** [reply]
Tal vez te refieras a algo distinto a estar registrado

Si acaso me comentas a qué te refieres con dos cursos y te aclaro la confusión.

**Comment by rramosp** [comment]
john, solo te veo registrado en un curso


On Thu, May 26, 2022 at 4:21 PM Jhon2507 ***@***.***> wrote:

> Yo no había visto la materia antes y me salen dos cursos
> Correo: ***@***.***
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/126#discussioncomment-2830391>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIP62JPND36NOYHHSE3VL7TN5ANCNFSM5WFUPZCA>
> .
> You are receiving this because you commented.Message ID: <rramosp/ai4eng.
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Comment by rramosp** [comment]
john, sólo te veo registrado en un curso
[image: image.png]

On Thu, May 26, 2022 at 4:21 PM Jhon2507 ***@***.***> wrote:

> Yo no había visto la materia antes y me salen dos cursos
> Correo: ***@***.***
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/126#discussioncomment-2830391>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIP62JPND36NOYHHSE3VL7TN5ANCNFSM5WFUPZCA>
> .
> You are receiving this because you commented.Message ID: <rramosp/ai4eng.
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Comment by Felipe-RA** [comment]
> miguel.rivas@udea.edu.co
> Tengo el curso duplicado y no puedo hacer los laboratorios

**Comment by rramosp** [comment]
ya lo tienes arreglado

On Tue, May 31, 2022 at 10:22 AM Felipe-RA ***@***.***> wrote:

> ***@***.***
> Tengo el curso duplicado y no puedo hacer los laboratorios
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/126#discussioncomment-2857144>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIPJ2HZV47QODLQ7N33VMYVEXANCNFSM5WFUPZCA>
> .
> You are receiving this because you commented.Message ID: <rramosp/ai4eng.
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Comment by Felipe-RA** [comment]
> julian.corredor@udea.edu.co
> me aparece 20221 y 2022.1 no me deja logearme en el colab

**Comment by rramosp** [comment]
ya lo tienes solucionado

On Wed, Jun 1, 2022 at 2:36 PM Felipe-RA ***@***.***> wrote:

> ***@***.***
> me aparece 20221 y 2022.1 no me deja logearme en el colab
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/126#discussioncomment-2866599>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIPIETKIH4M4HTNFJW3VM63UXANCNFSM5WFUPZCA>
> .
> You are receiving this because you commented.Message ID: <rramosp/ai4eng.
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Comment by lguerra98** [comment]
lukas.guerra@udea.edu.co
hola, tengo el curso dulicado por que el semestre pasado cancelé sin embargo pude hacer el primer lab pero para el segundo me esta apareciendo un error y no sé si sea por estar matriculado en dos cursos, gracias
![rt](https://user-images.githubusercontent.com/107425487/173401916-52483433-2fb3-4a8d-84cd-553d273d0626.png)

**Comment by rramosp** [comment]
hola lguerra, ese problema no es del curso duplicado, es que tienes que
ejecutar las celdas donde pones tu soluciones

On Mon, Jun 13, 2022 at 11:36 AM lguerra98 ***@***.***> wrote:

> ***@***.***
> hola, tengo el curso dulicado por que el semestre pasado cancelé sin
> embargo pude hacer el primer lab pero para el segundo me esta apareciendo
> un error y no sé si sea por estar matriculado en dos cursos, gracias
> [image: rt]
> <https://user-images.githubusercontent.com/107425487/173401916-52483433-2fb3-4a8d-84cd-553d273d0626.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/126#discussioncomment-2940228>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIL5C3T5KQSD273SRJDVO5PPJANCNFSM5WFUPZCA>
> .
> You are receiving this because you commented.Message ID: <rramosp/ai4eng.
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Reply by lguerra98** [reply]
ya las ejecute y luego estas, incluso me loggeo otra vez, pero veo un error como si la calificacion me estuviera pidiendo un "t1_actual" que eran del ejercicio anterior y el que estoy haciendo es el palindromo

**Reply by Felipe-RA** [reply]
Hola lguerra, 

Crea un post aparte y lo trabajamos desde ahí. Este post es para los cursos duplicados

Vimos que efectivamente si tenias un curso duplicado, aunque no esta asociado a la imagen que pusiste. Ya te eliminamos el duplicado

**Reply by lguerra98** [reply]
Hola creo que el error si se presentaba por los dos cursos que tenia, luego que lo borraron si me dejó calificar normalmente la tarea, igualmente gracias.

**Comment by Felipe-RA** [comment]
>> JUAN JOSÉ OSPINA RAMÍREZ
>> Buenas tardes, compañero yo todavía aparezco matriculado en la versión vieja del curso. juan.ospinar1@udea.edu.co

**Reply by Felipe-RA** [reply]
Listo, ya esta!

**Comment by natogourmet** [comment]
![image](https://user-images.githubusercontent.com/60532906/173425673-529d45c2-1efd-4f89-96df-97e32d51ec51.png)
Antes no me sucedía esto, nunca he tomado el curso previamente.
correo: stiven.guerra@udea.edu.co

**Reply by rramosp** [reply]
stiven, ya lo tienes arreglado

**Comment by rramosp** [comment]
eso es probablemente que has mezclado celdas de un notebook en otro. abre
de nuevo el notebook desde la página de curso y verifica que te estás
autenticando en el lab que es

[image: image.png]

On Mon, Jun 13, 2022 at 12:24 PM lguerra98 ***@***.***> wrote:

> ya las ejecute y luego estas, incluso me loggeo otra vez, pero veo un
> error como si la calificacion me estuviera pidiendo un "t1_actual" que eran
> del ejercicio anterior y el que estoy haciendo es el palindromo
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/126#discussioncomment-2940596>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIOT25BVUR5RJ4USG5DVO5VERANCNFSM5WFUPZCA>
> .
> You are receiving this because you commented.Message ID: <rramosp/ai4eng.
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Comment by rramosp** [comment]
ya está

On Mon, May 23, 2022 at 9:36 PM Felipe-RA ***@***.***> wrote:

> Buenas noches, me aparecen dos cursos en la plataforma y desde el colab no
> me deja avanzar la entrada. Mi correo es: ***@***.***
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/126#discussioncomment-2808517>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIM37KKYGUX476CQ43LVLQ6DVANCNFSM5WFUPZCA>
> .
> You are receiving this because you commented.Message ID: <rramosp/ai4eng.
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Comment by rramosp** [comment]
ya te eliminè la subscripción del semestre pasado

On Tue, May 17, 2022 at 9:49 PM MariaCamilaLM ***@***.***>
wrote:

> ***@***.***
>
> Tengo suscripciones duplicadas porque matriculé el curso el semestre
> anterior pero cancelé.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/126#discussioncomment-2772127>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZINJP244K2UJECLFFH3VKRLDTANCNFSM5WFUPZCA>
> .
> You are receiving this because you commented.Message ID: <rramosp/ai4eng.
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
